tests/test_unit.py

```diff
@@ -7,23 +7,33 @@
 # Copyright 2002-2019, Paul McGuire
 #
 #
-from unittest import TestCase, TestSuite, TextTestRunner
+from __future__ import absolute_import
+
 import datetime
-from pyparsing import ParseException
-import pyparsing as pp
-ppt = pp.pyparsing_test
 import sys
 from io import StringIO
+from unittest import TestCase
+
+import pyparsing as pp
+from pyparsing import ParserElement
+from examples.jsonParser import jsonObject
+from pyparsing import ParseException
+
+ppt = pp.pyparsing_test
+TestParseResultsAsserts = ppt.TestParseResultsAsserts
+
+try:
+    from tests.json_parser_tests import test1, test2, test3, test4, test5
+except Exception as e:
+    from json_parser_tests import test1, test2, test3, test4, test5
```

Comment:
@ptmcg   for some reason the command line and pyCharm use different import rules (absolute and relative respectively) . Until I find out how to fix that, I import both.

Reply:
In PyCharm, edit your run configuration for 'UnitTests' to set the parent directory as the working directory. This should then let you fix on absolute imports.

Reply:
fixed with https://github.com/pyparsing/pyparsing/pull/130/commits/dc70de952da931d655502c41a0db80fbd35e9571